### PR TITLE
4718 - Add Ids Block Grid

### PR DESCRIPTION
**Pull request by ericangeles**
**Explain the details for making this change. What existing problem does the pull request solve?**

Implements responsive layout for IDS Block Grid Component. Some functionalities from the enterprise are not yet here like paging, single and mixed selection.

**Related GitHub/Jira issue (required)**:

Closes https://github.com/infor-design/enterprise/issues/4811

**Steps necessary to review your pull request (required)**:

- Pull this branch, build, and run the app
- Go to http://localhost:4300/ids-block-grid
- Go check on different breakpoints to test the layout.
- Check the code implementations, tests (functional, e2e) and docs.

**Included in this Pull Request**:
- [x] An e2e or functional test for the bug or feature.
~~- [ ] A note to the changelog.~~

### CODE REVIEW

**Comment by tmcconechy** [comment]
Looks pretty good but a couple small comments...

1. If you make the browser wide - it looks like its not centered - can we make align="centered" the default.
2.Then after you do that.. You will see if you resize so 3 are on the first row. Wonder if there is a better way to lay it out? I see you used inline block but was thinking we would use css grid or flex in case of add numbered items I.E. can we do left align but also the whole thing centered within the page (but the tiles align left/right)

Visual image is ok but anticipating you will make a change so did not approve.

**Review by tmcconechy — DISMISSED** [review]

File: src/ids-block-grid/ids-block-grid.scss [hunk 1]
```diff
@@ -0,0 +1,38 @@
+@import '../../node_modules/ids-identity/dist/theme-uplift/tokens/web/theme-uplift-mixins.scss';
```
Comment (deep7102) [inline src/ids-block-grid/ids-block-grid.scss:1]: Think you do not need these uplift mixins already imported in `src/ids-base/ids-base`.

**Review by deep7102 — COMMENTED** [review]
Look good!,
are we going to do standalone-css example

**Review by tmcconechy — APPROVED** [review]
Looks great 👍🏻  We will need to merge https://github.com/infor-design/enterprise-wc/pull/47 with this. It might be easier if that goes in first and we merge it in.

I will help with that.

**Review by tmcconechy — APPROVED** [review]

**Pull request merged**